### PR TITLE
net-libs/gsoap: Fix incompatible integer to pointer conversion

### PR DESCRIPTION
**Pull request by listout**
and update EAPI 7 -> 8

Closes: https://bugs.gentoo.org/897870

### CODE REVIEW

File: net-libs/gsoap/files/gsoap-2.8.114-musl-strerror_r.patch [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Bug: https://bugs.gentoo.org/897870
```
Comment (thesamesam) [inline net-libs/gsoap/files/gsoap-2.8.114-musl-strerror_r.patch:1]: Please send this upstream, lgtm though

**Comment by thesamesam** [comment]
could you rebase the patch for .130 I just pushed & send upstream? thanks

**Comment by listout** [comment]
Sure.

**Comment by listout** [comment]
Upstream ticket crated: https://sourceforge.net/p/gsoap2/bugs/1314/

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**